src/mlpack/core/data/load_vec_impl.hpp

```diff
@@ -70,24 +68,22 @@ bool Load(const std::string& filename,
        * Now we can call the move operator, but it has to be the move operator
        * for Mat, not for Col.  This will avoid copying the data.
        */
-      *((arma::Mat<eT>*) &vec) = std::move(tmp);
+      *((arma::Mat<eT>*)&vec) = std::move(tmp);
       return true;
     }
   }
   else
   {
     // It's loaded as a column vector.  We can call the move constructor
     // directly.
-    *((arma::Mat<eT>*) &vec) = std::move(tmp);
+    *((arma::Mat<eT>*)&vec) = std::move(tmp);
```

Comment:
Hmm, maybe if we can show a simple reproducible example we can file a bug upstream.